pkg/webhook/validator.go

```diff
@@ -80,6 +80,8 @@ func NewWebhook(name string, mgr manager.Manager, validator Validator, apiType r
 	if err != nil {
 		logrus.Errorf("Error building webhook: %v", err)
 		os.Exit(1)
+	} else {
+		logrus.Info("Webhook started!")
```

Comment:
Can you change this line to `logrus.Info(name + " webhook started")`? We hit this function for several different controller types, so currently it looks like the webhook starts half a dozen times.